### ISSUE TITLE
Nesprávne OSM Attribution Guidelines na mapy.cz

### ISSUE THREAD

**Issue by zdila** [FreemapSlovakia/freemap-operations#21]
Stránka www.mapy.cz nespĺňa [OSM Attribution Guidelines](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines). Prevádzkovateľ stránky je Seznam.cz.

Ref. #19, #20.

**Comment by xsuchy** [comment]
Vlevo dole (pod měřítkem) je (c) Seznam a další. A když se klikne na to další, tak se vyroluje seznam dalších vlastníku a je tam i OSM. Resp. když jsem to zkoušel, tak tam byla jenom OSM.
IMHO je to trochu jinak než je zvykem, ale právnicky OK.
![Screenshot_20211105_164509](https://user-images.githubusercontent.com/85769/140538665-336c44a3-ec68-4882-a00c-892d327575c4.png)

**Comment by zdila** [comment]
@xsuchy pozri si [OSM Attribution Guidelines](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines) - attribution musi byt zobrazeny okamzite, nie az po pouzivatelskej akcii.

**Comment by xsuchy** [comment]
Koukám:
```
While the lower right corner is traditional, any corner of the map is acceptable. Alternatively, the attribution may be placed adjacent to the map or on a splash screen or pop-up shown when a user starts the app, device, website, etc.

You may use a mechanism to fade/collapse the attribution under certain conditions:

    immediately with a dismiss interaction, for example clicking an ‘x’ in the corner of a dialog
    automatically on map interaction such as panning, clicking, or zooming
    automatically after five seconds. This also applies to splash screens or pop-ups.

If the attribution has been collapsed, the user must still be able to find the licence information if they look for it, for example from an ‘(i)’ button in the corner of the map or an ‘About’ option in a menu.

If a splash screen or other method of attribution cannot link to the licence information, the link must be provided in an easily locatable part of the web page, software, or application (e.g. in the menu under “Data licences”). 
```
a přijde mi to OK.

Plus to mají v Licenčních podmínkách. https://napoveda.seznam.cz/cz/mapy/mapy-licencni-podminky/licencni-podminky-mapovych-podkladu/

Netvrdím, že by to nemohlo být lepší. Ale už jsem to s nimi jednou řešil a byl jsem odkázán přesně na tyhle dvě věci. Attribution Guidelines je sice hezká věc. Ale závázné je https://www.openstreetmap.org/copyright a to my přišlo, že splňují. Teda potažmo i ty guidelines, protože jsou plné SHOULD a MAY.
IMHO je to tak, že dobrý právník by to proti špatnému právníkovi vyhrál. A je jedno na které straně by kdo stál :)

**Comment by zdila** [comment]
Možno ti to príde OK, ale nesedí to s tým popisom, ktorý cituješ :-). Nepíšu to tam síce explicitne, ale jasne mi z toho vyplíva, že attribution sa musi zjaviť okamžite s mapou a až následne sú povolené mechanizmy na jeho skrytie.

**Comment by kudlav** [comment]
Ahoj, je tam mimo jiné toto:

> **[Requirements to fit within OSMF’s safe harbour](https://wiki.osmfoundation.org/wiki/Licence/Attribution_Guidelines#Requirements_to_fit_within_OSMF.E2.80.99s_safe_harbour)**
> * Attribution must be presented to anyone who uses, views, accesses, interacts with, or is otherwise exposed to the map or produced work. The attribution format should not require individuals to interact with the map or produced work to see the attribution.

Z toho plyne, že mapy.cz se nachází v šedé zóně licence, protože porušují **SHOULD**. A tedy pokud s nimi nebude rozumná domluva, není na ně žádná páka.

Zkusil bych mapy.cz kontaktovat a navrhnout, že místo _„a další”_ se rovnou zobrazí _„(c) OpenStreetMap_”, pokud je v sekci _„a další”_ pouze OSM. Z hlediska UI/UX je to v pohodě a při vypnutých leteckých snímcích je tam stejně jen to OSM.

Z mého osobního pohledu by si měla OSM sama nejprve ujasnit a sepsat jednoznačná pravidla. Na té wiki jsou zastaralé údaje a članky si vzájemně rozporují (_Licence/Licence and Legal FAQ_ a _Licence/Attribution Guidelines_). Napsal jsem email na legal-questions@osmfoundation.org, tak uvidím, jestli si to dají do kupy.

Na závěr jsem si to dovolil zaevidovat i sem: https://wiki.openstreetmap.org/wiki/Cs:Lacking_proper_attribution

**Comment by jose1711** [comment]
Zo strany mapy.cz to vidim na take trochu vych*anstvo typu: "neni to tam jasne uvedene, tak si to akoze vylozime po svojom." Pretoze upravit:
![obrázok](https://user-images.githubusercontent.com/1406222/144754818-3d8dbedf-ae20-41a5-9ef2-6d3f33e33a65.png)
tak, aby sa miesto "ďalšie" pri Seznam.cz zobrazilo "© OpenStreetMap" je podla mna najmenej.

**Comment by kudlav** [comment]
Ano, souhlasím. Zejména, když jediné, co se po kliknutí na " a další"
zobrazí, je © OpenStreetMap.

--
Vladan Kudláč

**Comment by matkoniecz** [comment]
<img width="404" height="45" alt="Image" src="https://github.com/user-attachments/assets/80db417f-fd2a-41e3-b383-8db39a359dea" />

legibility questionable, could and should be more clear (map style vs data) but attribution to OSM is there and not less prominent than self-attribution

**Comment by kudlav** [comment]
Hi, seems that it's displayed outside of Czechia, which is correct. Also, benches are missing on the web (they import them from OSM), and I don't know if there is any other POI which is imported from OSM within Czechia.
